Service/Controllers/RecommendationController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Management;
+using Management.Ports;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Service.Controllers
+{
+    [Route("api/recommendation")]
+    [ApiController]
+    public class RecommendationController : ControllerBase
+    {
+        private readonly RecommendationPort _recommendationPort;
+
+        public RecommendationController(RecommendationPort recommendationPort) 
+        {
+            _recommendationPort = recommendationPort ?? throw new ArgumentNullException(nameof(recommendationPort));
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> GetTopRecommendations()
+        {
+            try
+            {
+                return Ok();
+            }
+            catch (Exception)
+            {
+                return NotFound();
+            }
+        }
+
+        [HttpGet]
+        [Route("{countryCode}")]
```

Comment:
[minor]: Do we want to use something like `country/{countryCode}` as route or just use `{countryCode}`?
I'm fine with both ways.

Reply:
Oh I missed it, should have country

Reply:
changed